src/FormatFactory.cpp

```diff
@@ -45,10 +45,10 @@ FormatFactory::FormatFactory() {
     this->add_format<LAMMPSDataFormat>();
     this->add_format<SDFFormat>();
     this->add_format<CSSRFormat>();
+    this->add_format<GROFormat>();
 
     // VMD molfile plugins
     this->add_format<Molfile<DCD>>();
-    this->add_format<Molfile<GRO>>();
```

Comment:
Can you also remove the template instantiation in Molfile.cpp ?

Reply:
Whoops forgot about this.